File: food-recipe-core/Models/Recipe.cs
using System;
using System.Collections.Generic;

namespace recipe_food.Models
{
    public class Recipe
    {
        public Guid Id { get; set; }
        public string RecipeName { get; set; }
        public List<string> Ingridients { get; set; }
        public string Cooking { get; set; }
        public Uri ImageUrl { get; set; }
        public List<Tag> Tags { get; set; }
    }
}

File: food-recipe-core/src/app/models/Recipe.ts
import Tag from './Tag'

export class Recipe {

  constructor(data: any) {
    this.id = data.id;
    this.name = data.recipeName;
    this.ingridients = data.ingridients;
    this.cooking = data.cooking;
    this.imageUrl = data.imageUrl;
    this.tags = data.tags
  }

  id: string;
  name: string;
  ingridients: string[];
  cooking: string;
  imageUrl: string;
  tags: Tag[];
}

export default Recipe;
